x/subscription/keeper/grpc_query_next_to_month_expiry.go

```diff
@@ -0,0 +1,46 @@
+package keeper
+
+import (
+	"context"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/lavanet/lava/x/subscription/types"
+	timertypes "github.com/lavanet/lava/x/timerstore/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func (k Keeper) NextToMonthExpiry(goCtx context.Context, req *types.QueryNextToMonthExpiryRequest) (*types.QueryNextToMonthExpiryResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "invalid request")
+	}
+
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	gs := k.subsTS.Export(ctx)
```

Comment:
what is this hackery?
timer stores have a method to get the next closest timer

Reply:
done. see https://github.com/lavanet/lava/commit/7cdc3fdcd5c8fc2057c25de566506adbd3fa78e3